packages/sdk/src/endpoints/RelationshipsEndpoint.ts

```diff
@@ -29,4 +29,24 @@ export class RelationshipsEndpoint extends Endpoint {
     public async getAttributesForRelationship(relationshipId: string): Promise<ConnectorResponse<ConnectorAttributes>> {
         return await this.get(`/api/v2/Relationships/${relationshipId}/Attributes`);
     }
+
+    public async terminateRelationship(relationshipId: string): Promise<ConnectorResponse<ConnectorRelationship>> {
+        return await this.put(`/api/v2/Relationships/${relationshipId}/Terminate`);
+    }
+
+    public async requestRelationshipReactivation(relationshipId: string): Promise<ConnectorResponse<ConnectorRelationship>> {
+        return await this.put(`/api/v2/Relationships/${relationshipId}/Reactivate/Request`);
```

Comment:
```suggestion
        return await this.put(`/api/v2/Relationships/${relationshipId}/Reactivate`);
```

no need for that additional verb here